### PR TITLE
workspace.cx.cleanup

### PR DESCRIPTION
**Pull request by rotty3000**
- LPS-179580 Add default readinessProbe/livenessProbe to the generated LCP.json by the workspace plugin
- LPS-000000 rename to unmapped properties
- LPS-000000 move from legacy homePageURL to standarized baseURL
- LPS-000000 only add properties and typeSettings to frontend client extensions
- LPS-000000 add unmapped properties to all but frontend client extensions
- LPS-000000 SF
- LPS-182369 remove unused

### CODE REVIEW

**Comment by rotty3000** [comment]
ci:test:gradle-plugins

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/client/extension/ClientExtension.java [hunk 1]
```diff
@@ -73,14 +73,9 @@ public Map<String, Object> toJSONMap(String pid) {
...
-
-		set.forEach(
-			entry -> {
-				if (!pid.contains("CETConfiguration")) {
-					configMap.put(entry.getKey(), entry.getValue());
-				}
-			});
+		if (!Objects.equals("frontend", classification)) {
+			configMap.putAll(unmappedProperties);
+		}
```
Comment (drewbrokke) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/client/extension/ClientExtension.java:73]: Any reason we can't always add unmapped properties?

**Review by drewbrokke — COMMENTED** [review]
Just the one question, everything else looks good

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/client/extension/ClientExtension.java (hunk 1 above)
Comment (rotty3000) [reply modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/client/extension/ClientExtension.java:73]: hmm, I guess there's not a strong reason.

Comment (rotty3000) [reply modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/client/extension/ClientExtension.java]: fixed

**Review by gamerson — APPROVED** [review]
LGTM!

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/client/extension/ClientExtension.java (hunk 1 above)
Comment (drewbrokke) [reply modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/client/extension/ClientExtension.java]: My thought on this one is that it sets me up to try adjusting the CET side of things on portal's side and maybe remove the `typeSettings` concept from the workspace plugin completely, per our conversation last week.

**Comment by drewbrokke** [comment]
ci:test:gradle-plugins

**Comment by rotty3000** [comment]
ci:test:gradle-plugins

**Comment by rotty3000** [comment]
(this is based on bchan/master

**Comment by rotty3000** [comment]
ci:test:gradle-plugins

**Comment by drewbrokke** [comment]
Gonna close this one for now. @rotty3000 feel free to re-send whenever you're able.

**Pull request closed without merging**